pt/lessons/basics/sigils.md

```diff
@@ -131,6 +132,16 @@ iex> ~W/i love #{'e'}lixir school/
 ["i", "love", "\#{'e'}lixir", "school"]
 ```
 
+### NaiveDateTime
+
+Uma [NaiveDateTime](https://hexdocs.pm/elixir/NaiveDateTime.html) pode ser bem útil para criar rapidamente uam struct que representa um `DateTime` **sem** um timezone.
```

Comment:
Small typo in: rapidamente **uam** struct

Reply:
@anderkonzen fixed.